frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -11,13 +11,20 @@ const styles = {
   },
   header: {
     responseTypeIcons: {
-      color: "#00565E",
       margin: "0 8px",
       padding: 3,
       border: "1px solid #00565E",
       borderRadius: 4,
       cursor: "pointer"
     },
+    responseTypeIconsSelected: {
+      backgroundColor: "#00565E",
+      color: "white"
+    },
+    responseTypeIconsUnelected: {
```

Comment:
There is a little typo here: _Unelected_ instead of _Unselected_

Reply:
dropped this style entirely, as it is unneeded in 7e62505